extensions/vertx-web/deployment/src/main/java/io/quarkus/vertx/web/deployment/VertxWebProcessor.java

```diff
@@ -468,8 +479,16 @@ private String generateHandler(HandlerDescriptor desc, BeanInfo bean, MethodInfo
                     .setModifiers(ACC_PRIVATE | ACC_FINAL);
         }
 
-        implementConstructor(bean, invokerCreator, beanField, contextField, containerField);
-        implementInvoke(desc, bean, method, invokerCreator, beanField, contextField, containerField, transformedAnnotations,
+        FieldCreator validatorField = null;
+        if (desc.isProducedResponseValidated()) {
+            // Is the produced item needs to be validated, we inject the Validator
```

Comment:
"Is the produced item" -> "If the produced item"

Reply:
Fixed.